docs/puppeteer.md

```diff
@@ -100,3 +97,10 @@ await chrome.kill();
 
 })();
 ```
+
+### [Using Puppeteer in a custom gatherer](https://github.com/GoogleChrome/lighthouse/tree/master/docs/recipes/custom-gatherer-puppeteer)
+
+
+--------------
+
+**Note**: https://github.com/GoogleChrome/lighthouse/issues/3837 tracks the overall discussion for making Lighthouse work in concert with Puppeteer. Some things, like A/B testing the perf of UI changes, are tricky or not yet possible.
```

Comment:
I'm so curious where the `A/B testing the perf of UI changes` comes from :)

I wouldn't really expect this to be related to puppeteer integration.